JS/Lesson16/salih/script.js

```diff
@@ -116,27 +116,91 @@ for(const key in obj) {
 console.log(sum);
 console.log('- - - - - - - - - - - - - - - - - ');
 // 12. Write a loop that finds the average of an array of numbers.
-const numbers12 = [10, 20, 30, 40, 50];
+const numbers12 = [10, 20, 30, 40, 500];
+function sumArray(numbers){
+	let sum=0;
+	for (i=0; i< numbers.length; i++){
+		sum += numbers[i];
+	};
+	return sum/numbers.length;
+}
+console.log('Exercise 12.1: ', sumArray(numbers12));
 
 console.log('- - - - - - - - - - - - - - - - - ');
 // 13. Write a loop that prints out the sum of all even numbers between 1 and 50.
+function sumEvenNumbers(firstNum, secondNum){
+	let small, big, sum=0;
+	if (firstNum > secondNum){ 
+		small = secondNum;
+		big = firstNum;
+	} else {
+		small = firstNum;
+		big = secondNum;
+	};
+	console.log('11-The Small Number is ', small, '. The Big Number is ', big); //I used them to follow the code.
+	for (i=small; i < big+1; i++){
+		console.log('22-The Small Number is ', small, '. The Big Number is ', big, 'The i Number is ', i); //I used them to follow the code.
+		if (i % 2 == 0){
+			sum += small;
```

Comment:
I think the mistake is here. You need to use `i` as you're trying to loop trough small to the big number (including).